helm/charts/infra/values.yaml

```diff
@@ -5,17 +5,16 @@ nameOverride: ""
 fullnameOverride: ""
 
 global:
-
   image:
```

Comment:
Since the object is now empty, it's a good idea to set it to `{}`. Though I suggest keeping at least the `tag` and `pullPolicy` since they're still good defaults

Reply:
Thanks!

Does `repository` make sense to keep at all?